packages/thirdweb/src/exports/react-native/wallets.ts

```diff
@@ -0,0 +1,100 @@
+export {
```

Comment:
nit on the file structure:

I think it would make more sense to organize the exports like so:

`exports/<exportname>/default.ts <- all platforms that aren't separately specified (also the types come from this always)
`exports/<exportname>/web.ts` <- web only (optional)
`exports/<exportname/react-native.ts` <- rn only (optional)
`exports/<exportname/node.ts` <- node only (optional)


_something_ like that

Reply:
addressed in part 3